docs(bc): Update app declaration (#8)
bonitasoft/bonita-central-doc: modules/ROOT/pages/bc-app-declaration.adoc

In Bonita Central, we refer as application to the entity that will be built and run by Bonita. It is the content of a unique Git repository. 
One Runtime will be able to host several applications that share the same Business Data Model. 

== Prerequisites
. All Runtimes (nodes) linked to an application space should have the same version. 

> . To facilitate the data exchange, install Bonita Central in the same network as your Bonita Runtimes. If not, authorization to the network will be required. 

Reviewer: Is it the recommended deployment?

Author: which part? the versions or the "in the same network"?

> . To facilitate the data exchange, install Bonita Central in the same network as your Bonita Runtimes. If not, authorization to the network will be required. 

Reviewer: ```suggestion
. To facilitate the data exchange, install Bonita Central in the same network as your Bonita Runtimes. If not, since Bonita Central requires direct access to the nodes, you may have to configure your network. See xref:ROOT:network-configuration.adoc[network configuration] for details.
```

Author: Included in the PR.

. Have Bonita Runtime admin credentials 

== How to declare an application ? 

After installing Bonita Central, the first thing you will have to do is adding the applications you want to supervise. 
Nothing easier because you will be able to do it directly from the overview page. 

The only information you will have to provide is : 

* A name and a description for the application. Make it transparent both for technical and business users 
* Runtime end point - URL, IP adress - to access it 
* Connection credentials - Bonita Runtime admin username and password. This information will not be stored in Bonita Central. It will only be used to establish a connection with the engine.

[NOTE]
====
If one of the following authentication method is used : Basic authentication, LDAP, SSO, the following should be applied: 

* The username should be known in Bonita Runtime organization
* The username should not be assigned to a real member of your organization
* Assign admin authorization rights to the username
* The credentials will only be used for data exchange
====

=== Establishing a connection with Bonita Runtime 

> Once Bonita Central will connect to Bonita Runtime, an agent will be installed in Bonita Runtime. Also, a dedicated technical user only authorized to exchange data with Bonita Central will be then created in Bonita Runtime organization with the appropriate permissions. In order for those permissions to be taken into account, a restart of Bonita Runtime is required. 

Reviewer: ```suggestion
Once Bonita Central will connect to Bonita Runtime, an agent will be installed in Bonita Runtime. Also, a dedicated technical user will be created in Bonita Runtime organization with the appropriate permissions.  This user will be only authorized to exchange data with Bonita Central. 

[WARNING]
====
In order for those permissions to be taken into account, a restart of Bonita Runtime is required. 
====
```

Author: suggestion included.

> Once the card with the application in green status is displaied, healtcheck, monitoring and configuration for this application is available. 

Reviewer: ```suggestion
Once the card of the application has a green status, healthcheck, monitoring and configuration for this application are available. 
```

Author: suggestion included.
